deployment/demo-cfn-template.yaml

```diff
@@ -0,0 +1,189 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'DRIVER demo stack -- EC2 instances with Security Groups'
+Parameters:
+  KeyName:
+    Description: Name of an existing EC2 KeyPair to enable SSH access to the instance
+    Type: AWS::EC2::KeyPair::KeyName
+    ConstraintDescription: must be the name of an existing EC2 KeyPair.
+  AppInstanceType:
+    Description: Web app EC2 instance type
+    Type: String
+    Default: t2.medium
+    AllowedValues: [t2.nano, t2.micro, t2.small, t2.medium, t2.large]
+    ConstraintDescription: must be a valid EC2 instance type.
```

Comment:
It looks like there's not a good built-in solution for validating instance types. Could we add change this to `a valid EC2 instance type in the T2 family` and add `t2.xlarge` and `t2.2xlarge` to the list?